### DISCUSSION TITLE
vue2 项目中存在相同的公用函数要怎样处理？

### DISCUSSION THREAD

**Discussion by yesay** [empjs/emp#308] [General]
暴露组件的vue2项目  和 引用组件的vue2项目中都包含调用接口使用的公用函数
axios.js
......
export const getRequest = (url, params) => {
    let accessTokenRoad = getStore('accessTokenRoad');
    return axios({
        method: 'get',
        url: `${base}${url}`,
        params: params,
        headers: {
            'accessToken': accessTokenRoad
        }
    });
};
......

这种情况下，被引用的项目需要将 axios 也暴露出来吗

项目调试时，报如下的错误
![2022-08-30_092350](https://user-images.githubusercontent.com/3607509/187327219-8b48e554-495c-4d1d-b345-47a727dc0bbc.png)

**Comment by ckken** [comment]
试试 share 加上

**Comment by yesay** [comment]
是添加这个属性吗？
![2022-08-30_170229](https://user-images.githubusercontent.com/3607509/187396371-1b93a080-c325-4844-932f-77773c70f1d6.png)

**Comment by ckken** [comment]
是

**Comment by yesay** [comment]
已经添加了，需要将dependencies下所有的依赖都添加吗？
